build/css/live-editor.ui.css

```diff
@@ -390,35 +390,35 @@ button.hilite {
     box-shadow: 0 0 20px green;
 }
 
-/*
- * Contains styles for breakpoints and modifies the editor's appearance
- * when debugging.
- */
+dialog {
```

Comment:
The modal change is also something that I'd want heavy testing of. We had to make several modal changes in December when we got feedback about modals on small screens, ChromeBooks, etc.